pkg/lib/host-connector/util/aik_quote_verifer.go

```diff
@@ -129,8 +130,8 @@ func VerifyQuoteAndGetPCRManifest(decodedEventLog string, verificationNonce []by
 	secLog.Debugf("util/aik_quote_verifier:VerifyQuoteAndGetPCRManifest()  PCR manifest digest: %v", tpm2bDigest)
 
 	/* PART 2: TPMT_SIGNATURE
-	   Skip the first 2 bytes having the quote info size and remaining bytes, which includes signer info, nonce, pcr selection
-	   and extra data. So jump to TPMT_SIGNATURE
+	Skip the first 2 bytes having the quote info size and remaining bytes, which includes signer info, nonce, pcr selection
```

Comment:
I hope the code is formatted with go fmt and has no errors

Reply:
Formatted the code in vscode.

Reply:
use go fmt command, do not use IDE specific tool